templates/quickstart-crowd-dc.template.yaml

```diff
@@ -1133,9 +1133,9 @@ Resources:
           Value: !Ref AWS::StackId
         # NOTE: The leading COMMIT/TIMESTAMP are used to locate the position to update; see scripts/update-tags.py
         - Key: "atl:quickstart:commit-id"
-          Value: "COMMIT: 35bde08129b5af2b4dc805dcdc9c944f9d459ba0"
+          Value: "COMMIT: affb0cbc584fe198852c8aeba73859255e38de73"
```

Comment:
I am not sure if we should change the commits for these changes. They are just test related so I would keep them same.